docs/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
```

Comment:
Adding Enzyme to the docs and actually running the example code, makes the whole CI process for the docs much longer. We may also just hardcode the Enzyme example in the docs.

Reply:
Happy with that for now! We can later check how to automate tests if that's currently unfeasible